pages/common/vcsh.md

```diff
@@ -0,0 +1,27 @@
+# vcsh
+
+> Version Control System for `$HOME` - multiple Git repositories in `$HOME`.
+
+- Execute git command on managed repository:
+
+`vcsh {{repository_name}} {{git_command}}`
+
+- Initialize (empty) repository:
```

Comment:
We usually list commands in the order they are used. I think a user will `init` the repo first ? So we should mention init first, then others.

Reply:
Okay, i changed it from 'most used' to 'order of first use'